### ISSUE TITLE
Character Creator: Adding/Removing Animations Causes Issues

### ISSUE THREAD

**Issue by Kitakazi** [TheGameCreators/GameGuruRepo#4610]
Pretty sure this one was reported and fixed a while ago but guess its back and anyway here is how to reproduce. 

Create a zombie character in character creator. 
Clear all animations. 
Add 1 animation, I used ground_sit from the booster pack. 
Save your character. Then place them in your level and assign the Behavior 'Play Animation'
The ground_sit behavior is now gone and all the default animations have somehow reappeared. You are able to select any default animation from the drop-boxes even though you just created the character with only 1 animation. 

Even re-importing the character only shows 1 animation available, yet once assigned a behavior all the default ones reappear?

![image](https://user-images.githubusercontent.com/51464873/236100247-ae472ea0-e56e-4d0c-be32-0b633d52fbc3.png)

![image](https://user-images.githubusercontent.com/51464873/236100391-639ed466-b021-42a4-97a8-8ab6c427dacd.png)

**Comment by LeeBamberTGC** [comment]
@Kitakazi Yes this was tweaked a little from the first implementation. You need to tick the Default Animation option to use the animation provided by the original object:
![image](https://github.com/TheGameCreators/GameGuruRepo/assets/35840583/f6e4355b-015c-43a6-bdc8-e189845b76e6)

**Comment by Kitakazi** [comment]
Ok that does fix this problem. Thanks.

**Comment by Kitakazi** [comment]
Checking use default animations no longer fixes this problem. I made a character and removed all animations. Then I added some animations for them to be sitting. Once I exported out the character I cannot see those animations anywhere even when checking "Use default animations." Additionally if I try to create my own behavior with the Visual Behavior Editor and I select "Loop Animation" it only shows default solder animations to choose from, which I removed when from his list when I created him. It does not show any of the Sit animations I added. And since it's my own Behavior there isnt even a "Use Default Animations" option to begin with?
